package.json

```diff
@@ -40,6 +40,7 @@
     "faker": "^4.1.0",
     "helmet": "^3.16.0",
     "jest": "^24.7.1",
+    "joi": "^14.3.1",
```

Comment:
This has been deprecated. Should be `@hapi/joi` now - added it in one of my earlier PRs.